app/src/main/java/com/android/shelfLife/ui/overview/FirstFoodItem.kt

```diff
@@ -0,0 +1,101 @@
+package com.android.shelfLife.ui.newoverview
+
+import android.widget.Toast
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.android.shelfLife.R
+import com.android.shelfLife.model.foodFacts.FoodFacts
+import com.android.shelfLife.model.foodFacts.FoodUnit
+import com.android.shelfLife.model.foodFacts.Quantity
+import com.android.shelfLife.model.foodItem.FoodItem
+import com.android.shelfLife.ui.navigation.NavigationActions
+import com.android.shelfLife.ui.navigation.Route
+import com.android.shelfLife.ui.navigation.Screen
+import com.android.shelfLife.ui.utils.CustomButtons
+import com.android.shelfLife.ui.utils.CustomTopAppBar
+import com.android.shelfLife.ui.utils.FoodNameField
+import com.android.shelfLife.viewmodel.overview.FoodItemViewModel
+import kotlinx.coroutines.launch
+
+@Composable
+fun FirstFoodItem(
+    navigationActions: NavigationActions,
+    foodItemViewModel: FoodItemViewModel = hiltViewModel(),
+) {
+
+  val coroutineScope = rememberCoroutineScope()
+  val context = LocalContext.current
+
+  Scaffold(
+      topBar = {
+        CustomTopAppBar(
+            onClick = { navigationActions.goBack() },
```

Comment:
You need to reset the selectedfooditem (set it to null) and call foodItemViewModel.resetSearchStatus()